wordpress-mobile/WordPress-Android
# FluxC: Fix post list progress bar getting stuck

Fixes #5135. This PR fixes the mentioned issue with a reset however since we don't get the `onPostChanged` event, it's not possible to set the `mCanLoadMorePosts` flag. That shouldn't be a big deal though, once we do another fetch, it will be set to `false` if needed.

To test:
* Go into post list
* Trigger a manual fetch and immediately go into post detail
* Return back to post list and verify the progress bar is not visible anymore
